meshnet/exitnode/fw.go

```diff
@@ -224,6 +185,12 @@ func resetPeersTraffic(peers []TrafficPeer, commandFunc runCommandFunc) error {
 	}
 
 	for _, peer := range peers {
+		if peer.Routing {
```

Comment:
Actually it might be beneficial to move this to the very bottom of this function. As these rules are inserted into a different table, their insertion sequence to other rules doesn't matter. But moving their insertion to the very end would help with a small leak that is currently present in this code. At the moment the rules inserted just below allow full routing to the peer, until the `refreshPrivateSubnetsBlock` finishes execution (so while routing to the local network is forbidded, it is allowed for a very brief moment). By moving the insertion of masquerading rules to the bottom I think we could prevent this leak.